app/src/main/java/org/piwigo/helper/AccountHelper.java

```diff
@@ -1,6 +1,8 @@
 /*
  * Piwigo for Android
- * Copyright (C) 2016-2017 Piwigo Team http://piwigo.org
+ * Copyright (C) 2017 Raphael Mack http://www.raphael-mack.de
```

Comment:
This should be auto generated

Reply:
Do you know how to configure the android studio accordingly? So far only the generic header is generated on new file creation, but no author tag is added on modification. - my understanding is, that each editors name should be in the file header explicitly...

Reply:
I removed author info from the template intentionally.
Ultimately the code belongs to the Piwigo project.
A separate list of contributors could be maintained separately.

Reply:
I thought to remember, that it is wise from copyright and licensing point of view, to mention every contributor to a file in its header. Piwigo itself doesn't follow this as I have seen. Maybe a good compromise would be to describe in the readme, that the copyright is claimed by the piwigo project and its team but in case of need, the individual contributions and their authors are documented in the meta data of the commit and if needed for legal reasons the information shall be extracted their. With that it seems sufficient to give only the generic header in each file...

What do you think?